package.json

```diff
@@ -87,7 +87,8 @@
     "react-html-parser": "^2.0.2",
     "react-player": "^2.1.1",
     "react-responsive": "^8.1.0",
-    "simplebar-react": "^2.2.0"
+    "simplebar-react": "^2.2.0",
+    "styled-components": "^5.1.1"
```

Comment:
this must be a peer dependency otherwise we have two styled-components packages installed on the app